src/phoenix/trace/semantic_conventions.py

```diff
@@ -57,10 +57,13 @@ class MimeType(Enum):
     JSON = "application/json"
 
 
+LLM_FUNCTION_CALL = "llm.function_call"
```

Comment:
is function_call and OpenAI specific thing? I'm guessing it is.

Reply:
At the moment, it is OpenAI-specific.